doc/classes/@GlobalScope.xml

```diff
@@ -29,9 +29,6 @@
 		<member name="Geometry" type="Geometry" setter="" getter="">
 			[Geometry] singleton
 		</member>
-		<member name="GodotSharp" type="GodotSharp" setter="" getter="">
```

Comment:
I don't think this block should be removed in this commit. Could you please amend your commit with this removal... removed? :)